packages/fixie/src/main.ts

```diff
@@ -24,9 +24,24 @@ function showResult(result: any, raw: boolean) {
 }
 
 /** Deploy an agent from the current directory. */
-async function deployAgent(path?: string) {
-  const client = await FixieClient.Create(program.opts().url);
-  await FixieAgent.DeployAgent(client, path ?? process.cwd());
+function registerDeployCommand(command: Command) {
+  command
+    .command('deploy [path]')
+    .description('Deploy an agent')
+    .option('-e, --env <key=value>', 'Environment variables to set', (v, m: Record<string, string> | undefined) => {
```

Comment:
How do `-e` params interact with a `.env` file? Which will take precedence? It would be nice to be explicit about that in this description to the user.

Reply:
Added a longer description. Eventually I think we should move away from `.env` but these currently need to be specified with every `fixie deploy` so I won't push for that just yet.